contract/assembly/index.ts

```diff
@@ -292,82 +159,228 @@ export function sendTips(recipientExternalAccount: ExternalAccount, itemId: stri
     ),
     "New total tips amount exceeds allowance"
   );
-
   const donationAmount = u128.muldiv(Context.attachedDeposit, u128.from(100), u128.from(103));
   const feeAmount = u128.sub(Context.attachedDeposit, donationAmount);
-
   assert(
     u128.le(donationAmount, storage.get<u128>(MAX_AMOUNT_PER_TIP_KEY, u128.Zero)!),
     "Tips amount exceeds allowance"
   );
   assert(!u128.eq(feeAmount, u128.Zero), "Donation cannot be free");
-
-  const nearAccount = nearByExternal.get(recipientExternalAccount);
-
-  if (nearAccount) {
-    ContractPromiseBatch.create(nearAccount).transfer(donationAmount);
+  const autoclaimWallet = walletsForAutoclaim.get(accountGId);
```

Comment:
What will happen if an auto-claim account will be unlinked in CA-contract?
Should tips be transferred automatically to that account?
Should we make an additional inter-contract request to be aware in?

Reply:
No. For now, the owner has decided not to check the wallet for receiving tips after connecting it.